pkgs/top-level/all-packages.nix

```diff
@@ -5878,44 +5878,25 @@ in
   };
   purePackages = recurseIntoAttrs (callPackage ./pure-packages.nix {});
 
+  pythonNew = callPackage ./python-new-packages.nix {};
```

Comment:
One advantage of this name is that it is consistent with haskell: for Haskell, `haskell` also does not point to the compiler but there is `haskell.compiler.ghc7103` for example. Of course, with Haskell, it is more clear since the compiler and the language have different names.